### ISSUE TITLE
[Website] Schema Explorer: Ensures select element has an accessible name.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6430]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/explorer/Input.ChoiceSet.html 9/14/2021

### Application Name

Website

### Problem Description

[ADO 36108013](https://microsoft.visualstudio.com/OS/_workitems/edit/36108013)

**Repro Steps:**
1. Launch the above URL.
2. Navigate to header control ‘Schema Explorer’ using tab key and activate through enter key.
3. Run AI4W tool.
4. Observe the issue.

**Actual Result:**
Select element does not have an accessible name.

**Expected Result:**
Form controls must have accessible names.

### Card JSON

```JSON
No Card
```
